src/components/ConfigEditor/ConfigEditor.tsx

```diff
@@ -198,6 +206,45 @@ export const ConfigEditor: React.FC<Props> = (props: Props) => {
             });
           }}
         />
+
+      {config.secureSocksDSProxyEnabled && (
+          <>
+            <div className="gf-form-group">
+              <h3 className="page-heading">Secure Socks Proxy</h3>
+              <br />
+              <InlineField
```

Comment:
I think we want to use `Field` instead of `InlineField`

Reply:
I kept the `InlineField` component and removed the tooltip - adding the information in a div underneath the header so it looks like this (consistent with the other fields):
![Screenshot 2024-10-22 at 11 54 24 AM](https://github.com/user-attachments/assets/c249d5a7-905b-4aea-91ca-defa7ef4dda7)

If I change the `InlineField` component to `Field` and add the description in the component it ends up looking like this:
![Screenshot 2024-10-22 at 11 57 54 AM](https://github.com/user-attachments/assets/5bf3af1f-0716-4856-aae6-a52d57525cd5)

Wdyt?